.github/workflows/functional-test-cloud.yaml

```diff
@@ -74,7 +74,7 @@ env:
   # The Radius helm chart location.
   RADIUS_CHART_LOCATION: deploy/Chart/
   # The region for AWS resources
-  AWS_REGION: "us-west-2"
+  AWS_REGION: "us-west-1"
```

Comment:
We should make sure to cherry pick this into the release branch in another PR before merging this PR